Add blog post for Platform 1.4 release (#4789)
tenzir/tenzir: web/blog/tenzir-platform-v1.4/index.md

---

> title: "Tenzir Platform v1.4: Platform Alerts"

Reviewer: Now that the title is changed you likely need to specify the slug exactly, otherwise the links in the release will be hard to predict.

authors: [lava]
date: 2024-11-07
tags: [release, platform]
comments: true
---

[Tenzir Platform v1.4][github-release] adds platform alerts and
many fixes and improvements to the frontend.

![Tenzir Platform v1.4](./tenzir-platform-v1.4.excalidraw.svg)

[github-release]: https://github.com/tenzir/platform/releases/tag/v1.4.0

<!-- truncate -->

## Platform Alerts

The platform now supports monitoring individual nodes and triggering an
alert when the node stays disconnected for more than a configurable

> threshold

Reviewer 2: ```suggestion
threshold.
```

> This is useful for production nodes that are supposed to be online 24/7.

Reviewer 2: ```suggestion
This is useful for nodes that are supposed to be online 24/7.
```

> With alerts, it becomes easier

Reviewer 2: End of sentence?

> See [Platform CLI](/platform-cli) docs page for an overview and a usage

Reviewer 2: ```suggestion
See the [platform CLI](/platform-cli) docs page for an overview and a usage
```

example.

## Other Improvements

As usual there's also a slew of additional improvements and bug fixes
provided in this release:

>  * The tenant switcher is now significantly faster
>  * Added a new diagnostics drawer in dashboard page
>  * Fixed image downloads for charts

Reviewer 2: ```suggestion
 - The workspace switcher is now significantly faster
 - A new diagnostics drawer on the dashboard page makes warnings/errors more accessible
 - The image download of charts now works as expected
```
